pkgs/applications/networking/tcpkali/default.nix

```diff
@@ -0,0 +1,19 @@
+{stdenv, autoconf, automake, fetchurl, libtool, bison}:
+
+let version = "0.9"; in
+
+stdenv.mkDerivation {
+  name = "tcpkali-${version}";
+  src = fetchurl {
+    url = "https://github.com/machinezone/tcpkali/archive/v${version}.tar.gz";
+    sha1 = "f8400e6127773b9c3d87345133c454f26913a630";
+  };
+  buildInputs = [autoconf automake bison libtool];
+  preConfigure = "autoreconf -iv";
```

Comment:
Have you tried using `autoreconfHook`? Simply adding it to build inputs should obviate the need to pass `auto*` and `libtool` as inputs as well as doing manual `autoreconf`.